### ISSUE TITLE
Search: NVDA reads name of each combobox two times

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52015]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : NVDA2018.1.1 

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Navigate to Activity Bar and select "Search"(Cntrl+Shift+F) button.
3. Navigate to "Match Case" , "Match Whole Word" , "Use RegEx","Use Exclude settings and Ignore Files", "Replace All" comboboxes  using "Tab".
4. Search any text say "abc" in Search textbox and hit "Enter". No valid results are found which is read by NVDA.

**Actual:**
NVDA reads the name of each combobox(naviagted using "Tab" key) and also content "No results found. Review..." thrice. 

**Expected:**
NVDA should read the content and name of controls only once to the user when navigated using "Tab".

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screenreader users will get unnecessary extra information if controls and content are read thrice.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_ViewSearch_ScreenReader_ReadsThrice.pptx](https://github.com/Microsoft/vscode/files/2105692/A11y_VSCode_ViewSearch_ScreenReader_ReadsThrice.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Tab into search combo boxes - each time nvda reads it's title two times. I can repro only on win.

**Comment by roblourens** [comment]
Upstream https://github.com/nvaccess/nvda/issues/7841

**Comment by isidorn** [comment]
Let's close as dup of #51992
